src/createServer.js

```diff
@@ -1,3 +1,42 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase')
+const { getError } = require('./errors');
+
+function createServer() {
+  const server = http.createServer((request, response) => {
+    const requestURL = new URL(request.url, 'http://localhost:5700');
+    response.setHeader('Content-Type', 'application/json');
+
+    const originalText = requestURL.pathname.slice(1);
+    const targetCase = requestURL.searchParams.get('toCase');
+    const errors = getError(originalText, targetCase);
+
+    if (errors.errors.length) {
+      response.write(JSON.stringify(errors));
+      response.statusCode = 400;
+      response.end();
+
+      return;
+    }
+
+    const {
+      originalCase,
+      convertedText
+    } = convertToCase(originalText, targetCase);
+
+    response.statusCode = 200;
```

Comment:
You forgot to set the statusText:
"You should respond with status 200 and OK status text."